File: Selenium/Activities/Activity10.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver import ActionChains

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/drag-drop")
    action = ActionChains(driver)
    driver.maximize_window()
    print(f"Title of the page: {driver.title}")

    ball = driver.find_element(By.XPATH, "//img[@id='ball']")
    drop_zone1 = driver.find_element(By.XPATH, "//div[@id='dropzone1']")
    drop_zone2 = driver.find_element(By.XPATH, "//div[@id='dropzone2']")
		
    action.click_and_hold(ball).pause(2).move_to_element(drop_zone1).release().perform()
		
    drop_zone1_text = drop_zone1.text
    if drop_zone1_text == "Dropped!":
        print("Ball was dropped in Dropzone 1")
		
    action.click_and_hold(ball).pause(2).move_to_element(drop_zone2).release().perform()
		
    drop_zone2_text = drop_zone2.text
    if drop_zone2_text == "Dropped!":
        print("Ball was dropped in Dropzone 2")

File: Selenium/Activities/Activity10.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity10 {

	public static void main(String[] args) {
		WebDriver driver = new FirefoxDriver();
		Actions action = new Actions(driver);

		driver.get("https://training-support.net/webelements/drag-drop");
		driver.manage().window().maximize();
		System.out.println("Title of the page: " +driver.getTitle());
		WebElement ball = driver.findElement(By.xpath("//img[@id='ball']"));
		WebElement dropZone1 = driver.findElement(By.xpath("//div[@id='dropzone1']"));
		WebElement dropZone2 = driver.findElement(By.xpath("//div[@id='dropzone2']"));

		action.clickAndHold(ball).pause(1000).moveToElement(dropZone1).release().build().perform();

		String dropZone1Text = dropZone1.getText();
		if(dropZone1Text.equals("Dropped!")) {
			System.out.println("Ball was dropped in Dropzone 1");
		}

		action.clickAndHold(ball).pause(1000).moveToElement(dropZone2).release().build().perform();

		String dropZone2Text = dropZone2.getText();
		if(dropZone2Text.equals("Dropped!")) {
			System.out.println("Ball was dropped in Dropzone 2");
		}

		driver.close();
	}
}
